src/main/java/org/example/model/Response.java

```diff
@@ -1,9 +1,17 @@
 package org.example.model;
 
+import org.example.service.DeveloperService;
+
+import java.util.List;
+
 public class Response {
     private int code;
```

Comment:
The title should be the same as in the getters / setters

Reply:
fixed

Reply:
Still wrong